From fd1ef47e57c3ed527ca3ec87ede88f465c49b15b Mon Sep 17 00:00:00 2001
From: Zeng Heng <zengheng4@huawei.com>
Date: Fri, 26 May 2023 10:12:56 +0800
Subject: [PATCH 205/263] [Huawei] config: enable CONFIG_ARM64_AMU_EXTN to be
 compatible with AMU platform

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I73EPL

--------------------------------

The KunPeng newest released platform supports the feature of AMU,
and this configuration is backward compatible. This patch would
acts as a prepatch for the following patch series.

Signed-off-by: Zeng Heng <zengheng4@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 865a5f9050d1..7c53585e91f2 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -487,7 +487,7 @@ CONFIG_AS_HAS_CFI_NEGATE_RA_STATE=y
 #
 # ARMv8.4 architectural features
 #
-# CONFIG_ARM64_AMU_EXTN is not set
+CONFIG_ARM64_AMU_EXTN=y
 CONFIG_AS_HAS_ARMV8_4=y
 CONFIG_ARM64_TLB_RANGE=y
 # end of ARMv8.4 architectural features
-- 
2.17.1

